### PR TITLE
Improved stability and resource usage of Oxigraph when working with large graphs.

### PR DESCRIPTION
**Pull request by alivinco**
### Improved

- Improved stability and resource usage of Oxigraph when working with large graphs.

### Added

- Added `InstancesFromAvevaPiAF` step.

### Fixed

- UI bug fixes and improvements.

### CODE REVIEW

File: cognite/neat/graph/stores/graph_store.py [hunk 1]
```diff
@@ -226,40 +246,31 @@ def drop(self):
...
+                self.internal_storage_dir = Path(str(self.internal_storage_dir_orig) + "_" + str(time.time()))
+
             except Exception as e:
                 logging.error(f"Error dropping graph : {e}")
 
         elif self.rdf_store_type == RdfStoreType.GRAPHDB:
             r = requests.delete(f"{self.graph_db_rest_url}/repositories/{self.graph_name}/rdf-graphs/service?default")
             logging.info(f"Dropped graph with state: {r.text}")
 
+    def gc(self):
```
Comment (doctrino) [inline cognite/neat/graph/stores/graph_store.py:266]: Suggest renaming to 
```suggestion
    def garbage_collection(self):
```

File: cognite/neat/workflows/steps/lib/graph_extractor.py [hunk 2]
```diff
@@ -413,15 +414,162 @@ def convert_json_to_triples(
...
         convert_json_to_triples(json_data, root_node, "root", "root", None)
-
+        graph_store.graph.restart()  # restarting the graph to release the memory
         return FlowMessage(
             output_text=f"Data from source file imported successfully. Imported {nodes_counter} objects \
                             and {property_counter} properties ."
         )
+
+
+class InstancesFromAvevaPiAF(Step):
```
Comment (doctrino) [inline cognite/neat/workflows/steps/lib/graph_extractor.py:431]: Suggest adding a simple test for this one

**Review by doctrino — APPROVED** [review]
A very quick skimming through

File: cognite/neat/workflows/steps/lib/graph_extractor.py (hunk 2 above)
Comment (alivinco) [reply cognite/neat/workflows/steps/lib/graph_extractor.py:431]: Yes will do in separate PR (along with test for JSON importer) , as this requires to anonymize the data dump I have (can't be public) and it's a bit of work.

**Pull request merged**